common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -169,8 +169,12 @@ def get_student(self, student_uid):
         """
         url = self.generate_student_url(student_uid)
         payload = {'key_type': 'uid'}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        students = json.loads(response.content)
+        try:
+            students = []
+            response = requests.get(url, headers=self.request_headers, data=payload)
+            students = json.loads(response.content)
+        except requests.ConnectionError:
+            log.warning("Could not retrieve information about student identified by student_uid {student_uid}.".format(student_uid=student_uid))
```

Comment:
@macornwell This line is too long; please add some line breaks to make it shorter (less than 120 chars, cf. [EdX Python Style Guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization)).